commons/src/test/java/org/openmrs/eip/OauthTokenTest.java

```diff
@@ -13,21 +13,21 @@ public class OauthTokenTest {
 	@Test
 	public void isExpired_shouldReturnTrueIfAsOfDatetimeIsAfterExpiryDatetime() {
 		final long expiresAt = 1626898515;
```

Comment:
I don't think the changes to these tests are consistent with previous behavior and the test method naming, am starting to think that the incrementing logic should be moved to the OauthProcessor instead and keep the OuathToken as a data holder with no modifications.

Reply:
Ok...